W-20964138 Adding link to secure properties tool for Java 17 (#617)
mulesoft/docs-code-builder: modules/ROOT/pages/int-create-secure-configs.adoc

[[encryption-tool]]
=== Encryption with the MuleSoft Secure Properties Tool

> MuleSoft provides a tool for encrypting sensitive data depending on the Java version you use: 

Reviewer: ```suggestion
To encrypt sensitive data, use the MuleSoft tool for your Java version.
```
